arch/arm/configs/bcm2709_defconfig

```diff
@@ -1382,8 +1382,10 @@ CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
 CONFIG_RASPBERRYPI_POWER=y
 CONFIG_IIO=m
+CONFIG_IIO_BUFFER=m
```

Comment:
IIO_BUFFER is a boolean, and already enabled.

Reply:
It's probably selected by one or more of the drivers that depend on it. It's best to apply your source defconfig file then look at the content of .config to see what changes are required.